pkg/auth/jwt.go

```diff
@@ -140,3 +140,7 @@ func claimsToIdentity(c jwt.MapClaims) (i *Identity) {
 
 	return
 }
+
+func ClaimsToIdentity(c jwt.MapClaims) (i *Identity) {
```

Comment:
Just export `claimsToIdentity` fn, no need to have this intermediate fn..

Reply:
Sure